react/features/base/conference/actions.js

```diff
@@ -220,7 +220,15 @@ function _addConferenceListeners(conference, dispatch, state) {
 
     conference.on(
         JitsiConferenceEvents.USER_JOINED,
-        (id, user) => commonUserJoinedHandling({ dispatch }, conference, user));
+        (id, user) => {
```

Comment:
Isn't this executed only for mobile? do we need it?

Reply:
Hum, good point. Want me to drop it?